src/services/tasks/logs/index.ts

```diff
@@ -0,0 +1,33 @@
+import { HttpClient } from '../../http-client';
+import { ObjectId, OptionsWithRql } from '../../types';
+import { HttpInstance } from '../../../http/types';
+import { LogsService, TaskLog } from './types';
+
+export default (
+  client: HttpClient,
+  httpWithAuth: HttpInstance
+): LogsService => {
+  async function query(taskId: ObjectId, options: OptionsWithRql) {
+    const { data } = await client.get(
+      httpWithAuth,
+      `/${taskId}/logs${options?.rql || ''}`,
+      options
+    );
+    return data;
+  }
+
+  return {
+    async find(taskId: ObjectId, options?: OptionsWithRql): Promise<TaskLog[]> {
```

Comment:
Same issue here as the logs service implementation, hard coded pagination, so this is the limitation for now.